packages/inventory/src/EntityTable.js

```diff
@@ -181,7 +181,7 @@ class EntityTable extends React.Component {
                         { ...{
                             ...hasCheckbox && rows.length !== 0 ? { onSelect: this.onItemSelect } : {},
                             ...expandable ? { onCollapse: onExpandClick } : {},
-                            ...actions ? { actions } : {}
+                            ...actions && rows.length > 0 ? { actions } : {}
```

Comment:
looks like this could also be written `...actions && rows.length > 0 && { actions } ` 🤔

Reply:
Good point, I'll update it